client/gutenberg/editor/edit-post/components/layout/index.js

```diff
@@ -63,7 +62,6 @@ function Layout( {
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
 		<div className={ className }>
-			<DocumentTitle />
```

Comment:
This component has been removed. See https://github.com/WordPress/gutenberg/blob/master/docs/reference/deprecated.md#380